plugins/auth-keycloak/README.md

```diff
@@ -0,0 +1,74 @@
+# @amplication/plugin-auth-keycloak
+
+[![NPM Downloads](https://img.shields.io/npm/dt/@amplication/plugin-auth-keycloak)](https://www.npmjs.com/package/@amplication/plugin-auth-keycloak)
+
+Enable [KeyCloak](https://www.keycloak.org/) authentication on a service
+
+## Purpose
+
+This plugin adds the required code to use Passport Keycloak strategy on the generated NestJS application
+[**Keycloak**](https://www.keycloak.org/) is an authentication and authorization platform that provides the required tools to secure your applications and services.
+
+## Prerequisites
+
+- Running [Keycloak](https://www.keycloak.org/) instance. You can use the `docker-compose.dev.yaml`` file dor dev container.
```

Comment:
```suggestion
- Running [Keycloak](https://www.keycloak.org/) instance. You can use the `docker-compose.dev.yml`` file or dev container.
```
Also the `docker-compose.dev.yml` is created after the plugin is run. Due to this settings regarding keycloak cannot be provided before that leading to circular dependency.